src/class-supported-post-types.php

```diff
@@ -65,8 +97,11 @@ public function get_current_post_type(): string|false {
 			// phpcs:ignore WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.Security.NonceVerification.NoNonceVerification, WordPress.Security.NonceVerification.Recommended
 			if ( 'post.php' === $pagenow && ! empty( $_GET['post'] ) ) {
 				// phpcs:ignore WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.Security.NonceVerification.Recommended
-				$post_id   = absint( $_GET['post'] );
-				$post_type = get_post_type( $post_id );
+				$post_type = get_post_type( absint( $_GET['post'] ) );
+
+				if ( ! $post_type ) {
+					$post_type = '';
+				}
 			// phpcs:ignore WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.Security.NonceVerification.NoNonceVerification, WordPress.Security.NonceVerification.Recommended
 			} elseif ( 'post-new.php' === $pagenow ) {
```

Comment:
Same question I asked on PR wp-curate/85 ... does it make sense/is it possible to use a nonce here instead of bypassing?

Reply:
no, this is pulling from an admin url that does not have a nonce.